theano/compile/profiling.py

```diff
@@ -54,7 +54,7 @@ def _atexit_print_fn():
         destination_file = open(config.profiling.destination, 'w')
 
     for ps in _atexit_print_list:
-        if ps.fct_callcount >= 1 or ps.compile_time > 1:
+        if ps.fct_callcount > 1 or ps.compile_time > 1:
```

Comment:
I think it wasn't removed. Not being added back. Ignoring a file won't revert the changes done in the PR.